Frameworks/CoreGraphics/CGBitmapContext.mm

```diff
@@ -48,25 +51,40 @@ CGImageRef CGBitmapContextCreateImage(CGContextRef context) {
 }*/
 
 /**
- @Status Caveat
- @Notes tied to underlying implementation caveats in CGImageGetBitmapInfo.
- From CGImage.mm: "Only returns kCGImageAlpha information"
+@Status Interoperable
+@Notes
 */
 CGBitmapInfo CGBitmapContextGetBitmapInfo(CGContextRef context) {
-	CGImageRef imageRef = context->Backing()->DestImage();
-	CGBitmapInfo bitmapInfo = CGImageGetBitmapInfo(imageRef);
-	return bitmapInfo;
+    int ret;
+
+    if (context) {
+        ret = context->Backing()->DestImage()->Backing()->BitmapInfo();
```

Comment:
>Backing [](start = 23, length = 7)

Nit: probably makes sense to add BitMapInfo() method to Backing context so we don't have this chain here.